velox/exec/RowContainer.h

```diff
@@ -48,19 +50,15 @@ class Accumulator {
 
   void destroy(folly::Range<char**> groups);
 
-  /// Used only for spilling. Do not introduce other usages.
-  Aggregate* aggregateForSpill() const {
```

Comment:
Once `aggregateForSpill` is removed, seems `aggregate_` field is not needed anymore

Reply:
This is correct, hence, I removed it.

Reply:
Ah, I missed that change..., ignore this comment.

Reply:
No problem. Thank you for reviewing.